src/core/shape_inference/include/reverse_sequence_shape_inference.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Please use `shape_infer` function in the `ReverseSequence::validate_and_infer_types()` method